lib/services/elasticsearch.js

```diff
@@ -982,7 +986,8 @@ function getElasticsearchRequest(request, kuzzle) {
   }
 
   ['from', 'size', 'scroll', 'scrollId', 'refresh'].forEach(argumentName => {
-    if (typeof request.input.args[argumentName] !== 'undefined') {
+    if (typeof request.input.args[argumentName] !== 'undefined' &&
+        extraParams.includes(argumentName)) {
```

Comment:
:thinking:  Interesting point.
Your solution gives full responsibility on the consistency of the extra params to the caller. Which does make sense since we are in the Elasticsearch Service, which is supposed to be aware of the ES API.
Moreover, the hardcoded list is implicit, which means that some extra parameters specified by the caller may be stripped out without this being reflected by the contract specified in the function signature. Which is crappy.

I like your proposition.